gcloud/storage/acl.py

```diff
@@ -166,6 +166,21 @@ class ACL(object):
     """Container class representing a list of access controls."""
 
     _URL_PATH_ELEM = 'acl'
+    _PREDEFINED_QUERY_PARAM = 'predefinedAcl'
+
+    _PREDEFINED_ACLS = frozenset([
+        "private",
```

Comment:
I really wish we'd use double or single quote and not both. I'm unclear if you used double-quote on purpose because these are "text read by humans" or if it was incidental. As it were, I don't think this is text read by humans (at least not like a sentence is read by humans).

I don't want to be the small mind hobgoblin or whatever, but PEP8 says

> Pick a rule and stick to it.